purview/models/__init__.py

```diff
@@ -81,3 +37,62 @@ def replace(self, node, properties=None):   # pragma: no cover
 
 RelationshipManager.disconnect_all = disconnect_all
 RelationshipManager.replace = replace
+
+
+def story_flow(label):
+    if label == 'BugzillaBug':
+        return {
+            'uid_name': BugzillaBug.id_.db_property or BugzillaBug.id.name,
+            'forward_relationship': '{0}<'.format(
+                BugzillaBug.resolved_by_commits.definition['relation_type']),
+            'forward_label': DistGitCommit.__label__,
+            'backward_relationship': None,
+            'backward_label': None
+        }
+    elif label == 'DistGitCommit':
+        return {
+            'uid_name': DistGitCommit.hash_.db_property or DistGitCommit.hash.name,
+            'forward_relationship': '{0}<'.format(
+                DistGitCommit.koji_builds.definition['relation_type']),
+            'forward_label': KojiBuild.__label__,
+            'backward_relationship': '{0}>'.format(
+                DistGitCommit.resolved_bugs.definition['relation_type']),
+            'backward_label': BugzillaBug.__label__
+        }
+    elif label == 'KojiBuild':
+        return {
+            'uid_name': KojiBuild.id_.db_property or KojiBuild.id.name,
+            'forward_relationship': '{0}<'.format(KojiBuild.advisories.definition['relation_type']),
+            'forward_label': Advisory.__label__,
+            'backward_relationship': '{0}>'.format(KojiBuild.commit.definition['relation_type']),
+            'backward_label': DistGitCommit.__label__
+        }
+    elif label == 'Advisory':
+        return {
+            'uid_name': Advisory.id_.db_property or Advisory.id.name,
+            'forward_relationship': '{0}<'.format(
+                Advisory.triggered_freshmaker_event.definition['relation_type']),
+            'forward_label': FreshmakerEvent.__label__,
+            'backward_relationship': '{0}>'.format(
+                Advisory.attached_builds.definition['relation_type']),
+            'backward_label': KojiBuild.__label__
+        }
+    elif label == 'FreshmakerEvent':
+        return {
+            'uid_name': FreshmakerEvent.id_.db_property or FreshmakerEvent.id.name,
+            'forward_relationship': '{0}>'.format(
+                FreshmakerEvent.triggered_container_builds.definition['relation_type']),
+            'forward_label': 'Container'+KojiBuild.__label__,
+            'backward_relationship': '{0}>'.format(FreshmakerEvent.triggered_by_advisory
+                                                   .definition['relation_type']),
+            'backward_label': Advisory.__label__
+        }
+    elif label == 'ContainerKojiBuild':
```

Comment:
Could you make this like the following for consistency:
`elif label == 'Container{0}'.format(KojiBuild.__label__):`